react/src/views/Box/components/BoxCard.tsx

```diff
@@ -62,6 +62,27 @@ function BoxCard({
     return color;
   };
 
+  const product =
```

Comment:
can you please move this to BoxView. Data prep should not happen in the ui component, but in the container component, e.g. where you also infered the location from the shipmentDetail